storages/python/tests/test_bson_storage.c

```diff
@@ -0,0 +1,110 @@
+#include "minunit/minunit.h"
+
+#include "dlite.h"
+#include "dlite-macros.h"
+#include "dlite-storage-plugins.h"
+
+MU_TEST(test_save)
+{
+  DLiteStorage* s = NULL;
+
+  // Load JSON metadata
+  char* url = "json://" STRINGIFY(DLITE_ROOT) "/src/tests/test-entity.json?mode=r";
```

Comment:
In general we would like to keep lines shorter than 80 characters - makes them easy to read in a 80 character wide termical...

Reply:
I know, but I saw that it wasn't practiced throughout the DLite source code, so I didn't pay attention to it. But I'll fix it.

Reply:
Probably because there have been several authors. I think we should standardise on 80 character line length.